soc/arm/st_stm32/stm32f1/Kconfig.defconfig.stm32f100xx

```diff
@@ -0,0 +1,55 @@
+# ST Microelectronics STM32F100XX MCU
+
+# Copyright (c) 2020, Jonas Eriksson, Up to Code AB
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32F100CX || SOC_STM32F100RX || SOC_STM32F100VX
+
+config SOC
+	default "stm32f100xb"
+
+config NUM_IRQS
+	default 41
```

Comment:
Will depend on the XE or XB variant

Reply:
Correct, should be 60 for XE. Updated

Reply:
56

Reply:
Hmm, alright. The data sheet for XE (https://www.st.com/resource/en/datasheet/stm32f100rc.pdf) says 60, and I checked in the f103xe soc definition in the source, which also matches the data sheet (60/60). But I'm probably missing something, changing to 56. Feel free to enlighten me :-)

Reply:
This line is for XB, this number count the number of interrupts, starting from 0, so we take last number of defined IRQs (https://github.com/zephyrproject-rtos/hal_stm32/blob/566f4eb616f9e61bec01aca9918cace4b01e11b5/stm32cube/stm32f1xx/soc/stm32f100xb.h#L141) and add 1

Reply:
Ah, I see. Should be correct now.